indra/llcommon/tests/lldeadmantimer_test.cpp

```diff
@@ -162,64 +113,31 @@ void deadmantimer_object_t::test<4>()
-        ensure_equals("WOCM isExpired() returns true with 0.0 horizon time after bell ring",
-                      timer.isExpired(0, started, stopped, count), true);
+        CHECK_MESSAGE(timer.isExpired(0 == started, stopped, count, "WOCM isExpired() returns true with 0.0 horizon time after bell ring"), true);
         ensure_approximately_equals("WOCM ringBell has no impact on expired timer leaving stopped == started", started, stopped, 8);
-    }
-    {
-        // With cpu metrics
-        F64 started(42.0), stopped(97.0);
-        U64 count(U64L(8)), user_cpu(29000), sys_cpu(57000);
-        LLDeadmanTimer timer(0.0, true);
-
-        timer.start(0);
-        timer.ringBell(LLDeadmanTimer::getNow() + float_time_to_u64(1000.0), 1);
-        ensure_equals("WCM isExpired() returns true with 0.0 horizon time after bell ring",
-                      timer.isExpired(0, started, stopped, count, user_cpu, sys_cpu), true);
-        ensure_approximately_equals("WCM ringBell has no impact on expired timer leaving stopped == started", started, stopped, 8);
-    }
+    
 }
 
-
-// start(0) test - unexpired timer reports unexpired
-template<> template<>
-void deadmantimer_object_t::test<5>()
+TEST_CASE_FIXTURE(deadmantimer_test, "test_5")
 {
+
     {
         // Without cpu metrics
         F64 started(42.0), stopped(97.0);
         U64 count(U64L(8));
         LLDeadmanTimer timer(10.0, false);
 
         timer.start(0);
-        ensure_equals("WOCM isExpired() returns false after starting with 10.0 horizon time",
-                      timer.isExpired(0, started, stopped, count), false);
+        CHECK_MESSAGE(timer.isExpired(0 == started, stopped, count, "WOCM isExpired() returns false after starting with 10.0 horizon time"), false);
         ensure_approximately_equals("WOCM t5 - isExpired() does not modify started", started, F64(42.0), 2);
         ensure_approximately_equals("WOCM t5 - isExpired() does not modify stopped", stopped, F64(97.0), 2);
-        ensure_equals("WOCM t5 - isExpired() does not modify count", count, U64L(8));
-    }
-    {
-        // With cpu metrics
-        F64 started(42.0), stopped(97.0);
-        U64 count(U64L(8)), user_cpu(29000), sys_cpu(57000);
-        LLDeadmanTimer timer(10.0, true);
-
-        timer.start(0);
-        ensure_equals("WCM isExpired() returns false after starting with 10.0 horizon time",
-                      timer.isExpired(0, started, stopped, count, user_cpu, sys_cpu), false);
-        ensure_approximately_equals("WCM t5 - isExpired() does not modify started", started, F64(42.0), 2);
-        ensure_approximately_equals("WCM t5 - isExpired() does not modify stopped", stopped, F64(97.0), 2);
-        ensure_equals("WCM t5 - isExpired() does not modify count", count, U64L(8));
-        ensure_equals("WCM t5 - isExpired() does not modify user_cpu", user_cpu, U64L(29000));
-        ensure_equals("WCM t5 - isExpired() does not modify sys_cpu", sys_cpu, U64L(57000));
-    }
```

Comment:
-353 lines of code, +60. Lots of the test body has been removed from this file and others?